components/scream/data/scream_input.yaml

```diff
@@ -61,31 +81,31 @@ HOMME:
     partmethod        : 4
     cubed_sphere_map  : 0
     topology          : cube
-    test_case         : jw_baroclinic
     u_perturb         : 1
     rotate_grid       : 0
     ne                : 4
     statefreq         : 9999
     mesh_file         : /dev/null
     tstep             : 300
-    nu                : 7.0e+15
-    nu_div            : 1.0e+15
-    nu_p              : 7.0e+15
-    nu_q              : 7.0e+15
+    nu                : 3.4e-8 
+    nu_div            : -1 
+    nu_p              : -1
+    nu_q              : -1
     nu_s              : -1
     nu_top            : 2.5e+5
     se_ftype          : 0
     limiter_option    : 9
-    vert_remap_q_alg  : 1
-    hypervis_scaling  : 0
+    vert_remap_q_alg  : 10
+    hypervis_scaling  : 3.0
     hypervis_order    : 2
-    hypervis_subcycle : 2
-    hypervis_subcycle_tom : 0
-    theta_hydrostatic_mode : false
-    theta_advect_form : 0
-    tstep_type        : 10
-    dt_remap_factor   : 3
+    hypervis_subcycle : 1
+    hypervis_subcycle_tom : 1
+    theta_hydrostatic_mode : true
```

Comment:
Should we use non-hydro by default? I feel like stuff in the data folder might be used as a template by other people, without paying attention to some parameters. Besides, non-hydro might stress test more sections of the code?

Note: non hydro requires tstep_type 10.

Reply:
Yeah we can do that. This name list setup is for the one case that ran 31 days. And that one was hydrostatic. I've tried nongydrostatic but get errors. Although we may have addressed those recently. So probably worth trying again.

Reply:
Actually, we probably want both. We do this for SCREAMv0, we have a LR test that uses hydro, and a HR test that uses non-hydro. Note that Jim's yaml parser can query compset and we can set this appropriately. I vote we keep this as Aaron has it for this PR, and then add a subsequent PR with another v1 compset that tests the HR (non-hydro) config.

Reply:
Sounds good.